packages/transport-native/src/nativeUsb.ts

```diff
@@ -1,45 +1,24 @@
 import { WebUSB } from '@trezor/react-native-usb';
-import {
-    Transport as AbstractTransport,
-    AbstractApiTransport,
-    SessionsClient,
-    SessionsBackground,
-    UsbApi,
-} from '@trezor/transport';
+import { Transport as AbstractTransport, AbstractApiTransport, UsbApi } from '@trezor/transport';
 
 export class NativeUsbTransport extends AbstractApiTransport {
     // TODO: Not sure how to solve this type correctly.
     public name = 'NativeUsbTransport' as any;
 
-    private readonly sessionsBackground;
-
     constructor(params: ConstructorParameters<typeof AbstractTransport>[0]) {
         const { messages, logger } = params;
-        const sessionsBackground = new SessionsBackground();
-
-        const sessionsClient = new SessionsClient({
-            requestFn: args => sessionsBackground.handleMessage(args),
-            registerBackgroundCallbacks: () => {},
-        });
-
-        sessionsBackground.on('descriptors', descriptors => {
-            sessionsClient.emit('descriptors', descriptors);
-        });
 
         super({
             messages,
             api: new UsbApi({
                 usbInterface: new WebUSB(),
                 logger,
             }),
-            sessionsClient,
         });
-        this.sessionsBackground = sessionsBackground;
     }
 
     public listen() {
         this.api.listen();
-        this.sessionsBackground.dispose();
```

Comment:
hmmm why was this needed?

Reply:
Oof. It's because when I removed abort signal from session background, I wanted to keep it 1:1. But `sessionsBackground.dispose()` should've been part of `stop()`, not `listen()`, of course. So it's my mistake.